src/applications/edu-benefits/10203/content/directDeposit.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const directDepositDescription = () => {
```

Comment:
According to the React docs, "Always start component names with a capital letter." - https://reactjs.org/docs/components-and-props.html

Reply:
This is code follows the standard within vets-website for form content. Please see https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/disability-benefits/all-claims/content

Reply:
That makes sense. If that's the existing pattern, then it's often better to stay consistent.